### PR TITLE
[2.0] Add a Pillar to save the SUSE registry mirror and its certificate

### PR DESCRIPTION
**Pull request by ereslibre**
We'll be saving the suse registry mirror settings to the `Pillar` model so salt can read this settings without the need to port the internal api to 2.0.

### CODE REVIEW

**Comment by vitoravelino** [comment]
Migrated all the remaining stuff from master.

File: config/initializers/filter_parameter_logging.rb [hunk 1]
```diff
@@ -2,4 +2,4 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += [:password, :suse_registry_mirror_certificate]
```
Comment (ereslibre) [inline config/initializers/filter_parameter_logging.rb]: I think we want here `suse_registry_mirror_cert`

Comment (vitoravelino) [reply config/initializers/filter_parameter_logging.rb]: Yep.

**Comment by ereslibre** [comment]
Depends on https://github.com/kubic-project/velum/pull/420

**Comment by ereslibre** [comment]
[SUSE registry mirror backport](https://trello.com/c/QmGHalOd/563-suse-registry-mirror-backport)

**Comment by ereslibre** [comment]
Closed https://github.com/kubic-project/velum/pull/420, incorporating that commit here.

**Pull request merged**